ten-simple-rules-dockerfiles.Rmd

```diff
@@ -828,6 +833,7 @@ BM contributed to writing – review & editing.
 SJE contributed to conceptualisation and writing – review & editing.
 THe contributed to conceptualisation.
 THi contributed to writing – review & editing.
+BE contributed to visulisation and writing – review & editing.
```

Comment:
Thanks for my inclusion!

Would revising the overarching structure of the rules through the summary (figure) count as contributing to conceptualisation too?

Reply:
I'm not sure, but would tend to say not yet.

_Would you be so kind to raise this with the other authors, maybe via email?_ Not everyone has double-checked their new contributions statement anyway, so that would be helpful to trigger them looking and pinching in.

Reply:
I would tend to say no, conceptualization implies putting ideas together before anything exists, and creating the original thing. You’d have to time travel :)

Reply:
I think for me it would make sense to add reviewing and editing - I’ve been doing a lot of that too over this last week!

Reply:
Will add that.

Reply:
Sorry just getting to this. The CRediT role is defined as: 
> "Conceptualization – Ideas; formulation or evolution of overarching research goals and aims." 

This includes "evolution" which, unlike "formulation" would not require time travel ;) My thinking was that shaping/restructuring the rules aligns well with this, as that is *the* concept of the paper but happy to accept the consensus view. Should I still email about this?